spec/content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/check_service_retired_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_domain_file
+
+describe ManageIQ::Automate::Service::Retirement::StateMachines::Methods::CheckServiceRetired do
+  let(:admin) { FactoryGirl.create(:user_admin) }
+  let(:request) { FactoryGirl.create(:service_retire_request, :requester => admin) }
+  let(:service) { FactoryGirl.create(:service) }
+  let(:task) { FactoryGirl.create(:service_retire_task, :destination => service, :miq_request => request) }
+  let(:svc_task) { MiqAeMethodService::MiqAeServiceServiceRetireTask.find(task.id) }
+  let(:svc_service) { MiqAeMethodService::MiqAeServiceService.find(service.id) }
+  let(:vm) { FactoryGirl.create(:vm) }
+  let(:retired_vm) { FactoryGirl.create(:vm, :retired => true) }
+  let(:root_object) do
+    Spec::Support::MiqAeMockObject.new('service'             => svc_service,
+                                       'service_retire_task' => task,
```

Comment:
@d-m-u 
Everything fed into Automate is a Service Object. service_retire_task should point to svc_task instead of task